admin-dev/themes/new-theme/scss/components/_help_box.scss

```diff
@@ -1,25 +0,0 @@
-.help-box {
```

Comment:
Krystian asked a similar question, here's my answer:
The style of this .hint-box is like an alert with a thicker left border, which to me makes no sense because we are trying to harmonize the BO. Therefore, it has no impact for me if the hint box displays as an alert, because the Twig file bootstrap_4_layout.html.twig uses this markup: `<div class="alert alert-info">`,so it will render as an info alert. That's why I have removed this part.